x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/release_badge.ts

```diff
@@ -14,7 +14,7 @@ export const RELEASE_BADGE_LABEL: { [key in Exclude<RegistryRelease, 'ga'>]: str
     defaultMessage: 'Beta',
   }),
   experimental: i18n.translate('xpack.fleet.epm.releaseBadge.experimentalLabel', {
```

Comment:
the keys should be renamed or updated in `ja-JP.json` and `zh-CN.json` I think.

Reply:
Was told by @Bamieh that the values get marked as part of the translation process. I'm specifically not touching the keys here (as well as other variables) as mentioned in the PR description. Thanks for the check!

Reply:
okay, good to know, thanks.